ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -1021,8 +1027,10 @@ public void shouldPrintTypesList() {
           + "      \"memberSchema\" : {" + NEWLINE
           + "        \"type\" : \"STRING\"," + NEWLINE
           + "        \"fields\" : null," + NEWLINE
-          + "        \"memberSchema\" : null" + NEWLINE
-          + "      }" + NEWLINE
+          + "        \"memberSchema\" : null," + NEWLINE
+          + "        \"parameters\" : { }" + NEWLINE
```

Comment:
Is there a less crappy way of validating json returns? Load from file? (would also be good if it was order invariant, but i'm guessing in Java it will be fixed wrt to order in which the AST is walked?)

Reply:
a less crappy way would also be when we eventually upgrade to java 13 to get multiline strings https://openjdk.java.net/jeps/355